app/org/maproulette/controllers/api/UserController.scala

```diff
@@ -60,34 +60,71 @@ class UserController @Inject()(userDAL: UserDAL,
   }
 
   def getUser(userId: Long): Action[AnyContent] = Action.async { implicit request =>
-    this.sessionManager.authenticatedRequest { implicit user =>
-      if (userId == user.id || userId == user.osmProfile.id) {
-        Ok(Json.toJson(User.withDecryptedAPIKey(user)(crypto)))
-      } else if (user.isSuperUser) {
-        this.userDAL.retrieveByOSMID(userId, user) match {
-          case Some(u) => Ok(Json.toJson(User.withDecryptedAPIKey(u)(crypto)))
-          case None => this.userDAL.retrieveById(userId) match {
-            case Some(u) => Ok(Json.toJson(User.withDecryptedAPIKey(u)(crypto)))
-            case None => throw new NotFoundException(s"No user found with id '$userId'")
+    this.sessionManager.userAwareRequest { implicit user =>
```

Comment:
So the one problem I have with this is that if there is any kind of bug in this request a user's API key could be exposed by calling this method. I think what I would prefer to see is a separate API call to get basic user information. So for example another function call named "getBasicUser". This way even if this particular PR has no bug we protect ourselves from future bugs that could have bad security concerns.